src/custom-element.js

```diff
@@ -231,6 +231,9 @@ export function applyLayout_(element) {
     // Do nothing. Elements themselves will check whether the supplied
     // layout value is acceptable. In particular container is only OK
     // sometimes.
+  } else if (layout == Layout.FLEX) {
+    //Display the element as flex
+    element.style.display = 'flex';
```

Comment:
Element itself cannot have `display: flex` - only parent must have it. So, basically we don't need to do this here. This might the reason - hard to say.